mock-core-configs/etc/mock/eol/templates/mageia-7.tpl

```diff
@@ -63,9 +64,9 @@ skip_if_unavailable=False
 
 [updates-debuginfo]
 name=Mageia $releasever - {{ target_arch }} - Updates - Debug
-#baseurl=http://mirrors.kernel.org/mageia/distrib/$releasever/{{ target_arch }}/media/debug/core/updates/
+baseurl=https://distrib-coffee.ipsl.jussieu.fr/pub/linux/Mageia-archive/distrib/$releasever/{{ target_arch }}/media/debug/core/updates/
```

Comment:
This deserves inline comment :) using *.fr baseurl instead the mirror list.

Reply:
Should be fixed with 807eb40a60a68ddbd9bc4f2db15ffd23b7af994f.